test/extended/router/router.go

```diff
@@ -67,7 +67,7 @@ var _ = g.Describe("[sig-network][Feature:Router]", func() {
 			)
 		})
 
-		g.It("should serve routes that were created from an ingress", func() {
+		g.It("should serve routes that were created from an ingress [Serial]", func() {
```

Comment:
We need to serialize the IngressClass one.  This test is quite happy to work in parallel.  The IngerssClass one is the one that breaks clusters.

Reply:
@smarterclayton commit `046d3e5` serializes the correct k8s test.